acvm-repo/acvm/src/compiler/transformers/csat.rs

```diff
@@ -210,16 +210,16 @@ impl CSatTransformer {
                             }
                         } else {
                             // No more usable elements left in the old opcode
-                            opcode.linear_combinations = remaining_linear_terms;
                             break;
                         }
                     }
+                    opcode.linear_combinations.extend(remaining_linear_terms);
```

Comment:
Definitely want to do a master pull it looks like

Reply:
I believe 7 merges master so the diff should be cleaner in the next PR - sorry about that